prov/usnic/src/fi_ext_usnic.h

```diff
@@ -44,11 +44,19 @@
 
 #define FI_PROTO_RUDP (100U | FI_PROV_SPECIFIC)
 
-#define FI_EXT_USNIC_INFO_VERSION 1
+#define FI_EXT_USNIC_INFO_VERSION 2
+
+#define USDF_MAX_DEVNAME 16
+#define USDF_CAP_MAX 5
 
 /*
  * usNIC specific info
  */
+struct usdf_cap {
+	const char *capability;
+	int present;
```

Comment:
I'd prefix these members with `uc_`, which should still work even when the struct changes to `fi_usnic_cap`.